index.html

```diff
@@ -1619,9 +1619,9 @@ <h3>Add Proof</h3>
 <a>cryptographic suite</a> (<var>type</var>) and any other properties needed by
 the <a>cryptographic suite</a> type; an identifier for the
 <a>verification method</a> (<var>verificationMethod</var>) that can be used to
-verify the authenticity of the proof; and an [[!XMLSCHEMA11-2]] combined date
+verify the authenticity of the proof;  An [[!XMLSCHEMA11-2]] combined date
 and time string (<var>created</var>) containing the current date and time,
-accurate to at least one second, in Universal Time Code format. A
+accurate to at least one second, in Universal Time Code format, a
 <a href="#dfn-domain">security domain</a> (<var>domain</var>) and
```

Comment:
Hmm @TallTed, GitHub isn't letting me taking your changes. It gives me the error: "Applying suggestions on deleted lines is currently not supported.". I'm not sure what is up.

Reply:
*sighs* It's because my suggestion encompassed the changed (deleted & added) line 1624.  I've split my suggestion into two pieces which should both be applied.